WordPress/src/main/res/values/strings.xml

```diff
@@ -261,6 +261,10 @@
     <string name="multiple_status_label_delimiter" translatable="false">\u0020·\u0020</string>
     <string name="post_list_toggle_item_layout_cards_view">Switch to cards view</string>
     <string name="post_list_toggle_item_layout_list_view">Switch to list view</string>
+    <string name="post_waiting_for_connection_publish">Post will be published next time you device is online</string>
+    <string name="post_waiting_for_connection_pending">Post will be submitted for review next time your device is online</string>
```

Comment:
This wasn't in the mockup but I like it. I updated the test scenarios to use this message. 👍

Reply:
Ohh I missed it was missing in the test scenarios! It's already approved text from https://github.com/wordpress-mobile/WordPress-Android/issues/10224.